plugins/modules/aci_rest.py

```diff
@@ -447,7 +463,16 @@ def main():
         output_path = aci.params.get("output_path")
         if output_path is not None:
             with open(output_path, "a") as output_file:
-                json.dump([payload_output_file], output_file)
+                if rest_type == "json":
+                    json.dump([payload_output_file], output_file)
+                    output_file.write("\n")
+                elif rest_type == "xml":
+                    if aci.params.get("output_format") == "json" and HAS_XML_TO_DICT:
+                        json.dump([xmltodict.parse(payload)], output_file)
+                        output_file.write("\n")
+                    elif aci.params.get("output_format") == "xml" or not HAS_XML_TO_DICT:
```

Comment:
If aci.params.get("output_format") == "json" but HAS_XML_TO_DICT is false we should either fail or at least log a warning that we are proceeding with XML output_format even if user is not requesting it.